apps/full-site-editing/full-site-editing-plugin/blocks/template/index.js

```diff
@@ -10,7 +10,7 @@ import { __ } from '@wordpress/i18n';
 import edit from './edit';
 import './style.scss';
 
-registerBlockType( 'a8c/template-part', {
+registerBlockType( 'core/template', {
```

Comment:
This name is already taken in core :/ https://github.com/WordPress/gutenberg/blob/0fce12abc365ba85288bdb71b9115d598776fcb4/packages/block-library/src/template/block.json#L2, which results in the following console error: `Block "core/template" is already registered.`

Reply:
:oldmanyellsatcloud: 😄

Reply:
There is a `unregisterBlockType` function so we could run that just before re-registering our block.

Reply:
If we aim to eventually bring this into Core, I don't think unregistering Core blocks first would be an ideal approach. 😄